apps/code-examples/src/app/code-examples/layout/text-expand/inline/demo.component.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { SkyTextExpandHarness } from '@skyux/layout/testing';
+
+import { DemoComponent } from './demo.component';
+
+describe('Text expand inline demo', () => {
+  async function setupTest(
+    options: {
+      dataSkyId?: string;
+    } = {},
+  ): Promise<{
+    textExpandHarness: SkyTextExpandHarness;
+    fixture: ComponentFixture<DemoComponent>;
+    loader: HarnessLoader;
+  }> {
+    await TestBed.configureTestingModule({
+      imports: [DemoComponent, NoopAnimationsModule],
+    }).compileComponents();
+
+    const fixture = TestBed.createComponent(DemoComponent);
+    const loader = TestbedHarnessEnvironment.documentRootLoader(fixture);
+
+    const textExpandHarness: SkyTextExpandHarness = options.dataSkyId
+      ? await loader.getHarness(
+          SkyTextExpandHarness.with({
+            dataSkyId: options.dataSkyId,
+          }),
+        )
+      : await loader.getHarness(SkyTextExpandHarness);
+
+    return { textExpandHarness, fixture, loader };
+  }
+
+  it('should set up the text expand', async () => {
+    const { textExpandHarness, fixture } = await setupTest();
+
+    fixture.detectChanges();
```

Comment:
Harness methods shouldn't require all the `fixture.detectChanges()` and `fixture.whenStable()` calls. I think all you need is this first `fixture.detectChanges()` and everything else will just work.